config/config.go

```diff
@@ -165,16 +168,40 @@ func GetConfig(configPath *string) (*Config, error) {
 		Database: rawCfg.Database,
 	}, nil
 }
-
 func readConfigFile(configPath *string) (*rawConfig, error) {
 	var rawCfg rawConfig
-	file, err := ioutil.ReadFile(*configPath)
+	// read main config file.
+	mainFileByte, err := ioutil.ReadFile(*configPath)
+	if err != nil {
+		return nil, errors.Wrap(err, "read config file")
+	}
+	if _, err := toml.Decode(string(mainFileByte), &rawCfg); err != nil {
+		return nil, errors.Wrap(err, "read config file")
+	}
+	// if no sub config file
+	if rawCfg.Include == "" {
+		return &rawCfg, nil
+	}
+	// read sub config files.
+	dir, err := ioutil.ReadDir(rawCfg.Include)
```

Comment:
```suggestion
	dir, err := ioutil.ReadDir(path.Join(*configPath, rawCfg.Include))
```

Reply:
I think `configPath` is the name of the main configuration file. If it is written in this way, the path built through `ioutil.ReadDir(path.Join(*configPath, rawCfg.Include))`will be wrong.

Reply:
Do you mean the `rawCfg.Include` is relative path to the executable file instead of main configuration file?

Reply:
yes

Reply:
Sounds reasonable, it is also good for include an absolute path.